admin_manual/ai/app_context_chat.rst

```diff
@@ -9,7 +9,7 @@ Context Chat is an :ref:`assistant<ai-app-assistant>` feature that is implemente
  * the *context_chat* app, written purely in PHP
  * the *context_chat_backend* ExternalApp written in Python
 
-Together they provide the ContextChat text processing tasks accessible via the :ref:`Nextcloud Assistant app<ai-app-assistant>`.
+Together they provide the ContextChat *text processing* and *search* tasks accessible via the :ref:`Nextcloud Assistant app<ai-app-assistant>`.
```

Comment:
Should we find a more elegant formulation than "text processing"?
No idea how to make this nicer but I feel like it could be improved. I don't mind keeping it like it is.